blocks/api/test/parser.js

```diff
@@ -548,187 +568,196 @@ describe( 'block parser', () => {
 	} );
 
 	describe( 'parse()', () => {
-		it( 'should parse the post content, including block attributes', () => {
-			registerBlockType( 'core/test-block', {
-				attributes: {
-					content: {
-						type: 'string',
-						source: 'text',
+		// run the test cases using the PegJS defined parser
+		testCases( parsePegjs );
```

Comment:
Can we put those test cases in their own `describe` blocks to make it easier to debug which version of the parser failed?